src/api/files/repository.js

```diff
@@ -19,6 +19,24 @@ export async function create(fileStatus) {
   logger.info(`Created file status for file ID ${fileStatus.fileId}`)
 }
 
+/**
+ * Adds a form to the Form Store
```

Comment:
Method and param description don't look correct

Reply:
Good spot, thanks. I nicked the skeleton code from Forms Manager, forgot to change it.